example.exs

```diff
@@ -136,7 +136,20 @@ sheet6 =
   # nest further
   |> Sheet.group_cols("C", "D")
 
+# Images
+sheet7 = %Sheet{
+  name: "Images",
+  rows: List.duplicate(["A", "B", "C", "D", "E"], 5)
+}
+
+sheet7 =
+  sheet7
+  |> Sheet.insert_image(0, 5, "ladybug-3475779_640.jpg")
```

Comment:
Can you verify that this is a picture we can actually use in an open source project?
If required, can you add an attribution notice (not a legal expert, but maybe just a ladybug-3475779_640.jpg.NOTICE txt file or something)